### DISCUSSION TITLE
Being rejected because the web camera not working properly

### DISCUSSION THREAD

**Discussion by INmahi** [community/community#57471] [GitHub Education]
### Hi everyone,

I have been trying to apply  for the student pack since the last week. I have a school id which have my name and session and picture on one side and validity date on the back side.
Moreover when I try to capture the proof image, the web camera only works with the front camera which is blury and hard to capture with.
I can manage transcript or other proofs that have date and my name on the same page but am Worried about the fact that I can neither upload the picture (image attached) nor take take clear picture with back camera.
My College website have my name and profile on it.
![Screenshot_20230608-015638~2](https://github.com/community/community/assets/71538842/cd06f8de-da48-436f-9643-e6a7a2f080c9)

**Comment by samus-aran** [comment]
Hi @INmahi 

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.
